src/context/AirDBContext.tsx

```diff
@@ -2,19 +2,29 @@ import Airtable from 'airtable'
 import React, { createContext } from 'react'
 
 export const AirDBContext = createContext<{
-  postAirDB: Function
-  getAirDB: Function
-  updateAirDB: Function
+  handleLogin: Function
+  handleAddTest: Function
+  handleAddQuestion: Function
+  updateTestQuestions: Function
+  handleQuestionAnswer: Function
+  createInitialScore: Function
+  handleLogout: Function
+  handleSetScore: Function
   tests: any[]
   questions: any[]
   loading: boolean
   scores: any[]
   users: any[]
   testInstances: any[]
 }>({
-  postAirDB: () => {},
-  updateAirDB: () => {},
-  getAirDB: () => {},
+  handleLogin: () => {},
```

Comment:
This is totally a nit pick. If you want, you could have a variable called `voidFunction` or even `emptyFunction` that would communicate what all these initial values mean. I do like that you don't have to check whether the function exists before calling it. 👍